### PR TITLE
Broker Zero-Nesting Sample and Code Changes

### PR DESCRIPTION
**Pull request by pkanher617**
This PR adds a sample showing a Zero-nesting case for the broker as well as minor changes to get this sample working with our library.

### CODE REVIEW

File: lib/msal-browser/src/broker/client/BrokerClientApplication.ts [hunk 1]
```diff
@@ -114,7 +114,7 @@ export class BrokerClientApplication extends ClientApplication {
 
         // @ts-ignore
         clientMessage.source.postMessage(brokerHandshakeResponse, clientMessage.origin);
-        this.logger.info(`Sending handshake response: ${brokerHandshakeResponse}`);
+        this.logger.info(`Sending handshake response: ${JSON.stringify(brokerHandshakeResponse)}`);
```
Comment (tnorling) [inline lib/msal-browser/src/broker/client/BrokerClientApplication.ts:117]: May contain PII?

**Review by tnorling — APPROVED** [review]

File: lib/msal-browser/src/broker/client/BrokerClientApplication.ts (hunk 1 above)
Comment (pkanher617) [reply lib/msal-browser/src/broker/client/BrokerClientApplication.ts:117]: Removing this log message.

File: lib/msal-browser/src/broker/client/EmbeddedClientApplication.ts [hunk 2]
```diff
@@ -166,14 +167,15 @@ export class EmbeddedClientApplication {
                     if (brokerHandshakeResponse) {
                         clearTimeout(timeoutId);
                         this.logger.info(`Received handshake response: ${JSON.stringify(brokerHandshakeResponse)}`);
+                        window.removeEventListener("message", onHandshakeResponse);
                         resolve(brokerHandshakeResponse);
                     } else {
-                        this.logger.warning(`Message is not handshake response: ${message}`);
+                        this.logger.warning(`Message is not handshake response: ${JSON.stringify(message.data)}`);
```
Comment (jasonnutter) [inline lib/msal-browser/src/broker/client/EmbeddedClientApplication.ts:173]: I would split this into two messages, with the data included in a verbose message.

Comment (jasonnutter) [reply lib/msal-browser/src/broker/client/EmbeddedClientApplication.ts:173]: Also, may contain PII?

Comment (pkanher617) [reply lib/msal-browser/src/broker/client/EmbeddedClientApplication.ts:173]: done.

**Pull request merged**